core/scripts/Voting.js

```diff
@@ -540,4 +566,9 @@ run = async function(callback) {
 };
 
 run.VotingSystem = VotingSystem;
+// TODO(ptare): Another way to allow modifying SUPPORTED_IDENTIFIERS for unit testing purposes is to wrap all the fetch*
```

Comment:
I think the way this is done now is probably the most sensible way to do it for now. If we need a more sophisticated approach for different networks, etc, we can implement that when we need it.